src/lollipopChart.js

```diff
@@ -385,6 +411,12 @@ var LollipopChart = function (selection) {
     if(chart.isBadDatum(d)) return svgHeight + lollipopRadius;
     return svgHeight - chart.generateLollipopHeight(d);
   }
+  
+  function generateLollipopLabelAxisX(d, i) {
+    var barX = generateBarX(d, i);
+    
+    return barX - barGap * (1/16);
```

Comment:
Does the 1/16 value work no matter the amount of data given to the chart? For example if there were 5 bars instead of 8?